### ISSUE TITLE
[Feature request] Add GPM (general purpose mouse) support

### ISSUE THREAD

**Issue by veronikazaglotova** [akermu/emacs-libvterm#444]
is:issue gpm shows nothing so I made this one.
It would be nice to have GPM support in emacs-libvterm. I really like this feature for htop specifically, because for htop it's faster to use the mouse when changing settings.
Also, it can be useful even for people who don't use the mouse. Make a minor mode (or use copy-mode for that?), in which you can move the Emacs cursor in any place on the screen, then RET uses GPM to send a mouse click at the place where the Emacs cursor is. This way, you can use any Emacs shortcut to move around instead of using the arrow keys in htop or using hjkl in vifm/nnn/mc/etc.

**Comment by Sbozzolo** [comment]
At the moment, this is quite challenging to implement. I am not even sure what is the state of mouse support in `libvterm`.
In the (very) long run, I think that we would like to have this, but I don't see it coming any time soon.

**Comment by veronikazaglotova** [comment]
Yeah, I found out about libvterm having no mouse support. It's very sad :(

**Comment by CyberShadow** [comment]
- See also: #15
